src/core/Icon/index.stories.tsx

```diff
@@ -113,7 +113,7 @@ const IconWrapper = styled("div")`
       }
 
       span {
-        color: ${colors?.gray[400]};
+        color: ${colors?.gray[600]};
```

Comment:
Darkened for color contrast.

This is just for a story, so doesn't really matter. But may as well still pass the checks.